### DISCUSSION TITLE
Caps lock does not work for LXQT

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#244] [Q&A] [answered]
I'd like to report an error with your LXQT image. On my old ASUS K55A laptop, the caps lock does not work on spiral linux LXQT. I tested the Debian non-free LXQT live ISO and the caps lock does work there. I used a thumb drive to test this.

**Comment by geckolinux** [comment, marked as answer]
Hi there, that's by design. The Caps Lock key works to switch keyboard layouts for multilingual users. You can disable it here:

![Screenshot from 2023-03-25 14-29-29](https://user-images.githubusercontent.com/2692138/227737657-f8f7a60f-26fe-43af-910e-4e159e2a2181.png)

**Reply by birds-swim** [reply]
Thank you for replying!

Question: are those the typical defaults or are those your changes?

**Reply by geckolinux** [reply]
Hi there @birds-swim , those are my changes.

**Reply by birds-swim** [reply]
Okay, thank you for the clarification.
